proposals/0036-kind-signatures.rst

```diff
@@ -77,7 +77,7 @@ Proposed Change Specification
    is done before ever examining the full declaration, just like how GHC treats type
    signatures.
 
-   Associated types may be given kind signatures within their classes.
+   Associated types may not be given kind signatures.
```

Comment:
It would be helpful to give a forward reference to the Costs and Drawbacks section here, since someone not in the know will likely find this sentence quite surprising without any context to explain why this choice was made.